<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KGeneralLoanInfoPage</class>
  <widget class="QWidget" name="KGeneralLoanInfoPage">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>675</width>
        <height>494</height>
      </rect>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QLabel" name="qLabel1">
          <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
            <string>Please select the required options. Please use the &lt;b&gt;What's this?&lt;/b&gt; feature to see more information about the items.</string>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QGridLayout">
              <item row="0" column="0">
                <widget class="QLabel" name="textLabel1">
                  <property name="text">
                    <string>What is the type of the loan?</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="KComboBox" name="m_loanDirection">
                  <item>
                    <property name="text">
                      <string>I am borrowing money</string>
                    </property>
                  </item>
                  <item>
                    <property name="text">
                      <string>I am lending money</string>
                    </property>
                  </item>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="textLabel2">
                  <property name="text">
                    <string>Who is the payee/payer of the loan?</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="KMyMoneyPayeeCombo" name="m_payee">
                  <property name="whatsThis" stdset="0">
                    <string>Enter the name of the person or bank you owe money or who owes you money.

If the name does not exist within KMyMoney's database, you are asked if you want to create it.</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="0">
                <widget class="QLabel" name="textLabel1_2">
                  <property name="text">
                    <string>Did you make/receive any payments yet?</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="2" column="1">
                <widget class="KComboBox" name="m_anyPayments">
                  <property name="whatsThis" stdset="0">
                    <string>Were there any payments for this loan whether they are entered into KMyMoney or not?

Note: Payments made to obtain the loan (e.g. Disagio) are not considered as payments in this context.</string>
                  </property>
                  <item>
                    <property name="text">
                      <string comment="@item no payments were entered for a loan">No</string>
                    </property>
                  </item>
                  <item>
                    <property name="text">
                      <string comment="@item payments were entered for a loan">Yes</string>
                    </property>
                  </item>
                </widget>
              </item>
              <item row="3" column="0">
                <widget class="QLabel" name="textLabel3">
                  <property name="text">
                    <string>Which payments do you want to record?</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="3" column="1">
                <widget class="KComboBox" name="m_recordings">
                  <property name="whatsThis" stdset="0">
                    <string>&lt;qt&gt;Select which transactions should be recorded.

&lt;b&gt;All payments&lt;/b&gt; allows you to enter all payments made for this loan.

The option to &lt;b&gt;start from the beginning of the current year&lt;/b&gt; is meant for loans that have already been active for a longer period of time, and for which you do not want to enter all past transactions.
&lt;/qt&gt;</string>
                  </property>
                  <item>
                    <property name="text">
                      <string>All payments</string>
                    </property>
                  </item>
                  <item>
                    <property name="text">
                      <string>Start with this year's payments</string>
                    </property>
                  </item>
                </widget>
              </item>
              <item row="4" column="0">
                <widget class="QLabel" name="textLabel1_3">
                  <property name="text">
                    <string>Balance before start of recording</string>
                  </property>
                  <property name="whatsThis" stdset="0">
                    <string>Enter the ending balance found on the statement that is the last one before you want to start recording this loan in KMyMoney.</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="4" column="1">
                <widget class="AmountEdit" name="m_openingBalance">
                  <property name="whatsThis" stdset="0">
                    <string>You have selected to record only payments from the beginning of this year. Since prior transactions will not be recorded, you need to supply the balance of the loan on January 1st of this year.</string>
                  </property>
                </widget>
              </item>
              <item row="5" column="0">
                <widget class="QLabel" name="textLabel5">
                  <property name="text">
                    <string>Payment frequency</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="5" column="1">
                <widget class="KMyMoneyFrequencyCombo" name="m_paymentFrequency">
                  <property name="focusPolicy">
                    <enum>Qt::TabFocus</enum>
                  </property>
                </widget>
              </item>
              <item row="6" column="0">
                <widget class="QLabel" name="textLabel1_4">
                  <property name="text">
                    <string>Interest Compounding frequency</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="6" column="1">
                <widget class="KMyMoneyFrequencyCombo" name="m_compoundFrequency">
                  <property name="whatsThis" stdset="0">
                    <string>Select the frequency with which the interest is compounded. If uncertain, select the same as the payment frequency. Consult your loan contract for details.</string>
                  </property>
                </widget>
              </item>
              <item row="7" column="0">
                <widget class="QLabel" name="textLabel2_2">
                  <property name="text">
                    <string>Due date of first payment to be recorded</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="7" column="1">
                <widget class="KMyMoneyDateInput" name="m_firstPaymentDate">
                  <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                  </property>
                  <property name="whatsThis" stdset="0">
                    <string>Depending on the answer to &lt;b&gt;What do you want to record?&lt;/b&gt; this field means the following:&lt;p&gt;

&lt;b&gt;All payments&lt;/b&gt;&lt;br&gt;
Enter the due date of the very first payment&lt;p&gt;

&lt;b&gt;Start with this year's payments&lt;/b&gt;&lt;br&gt;
Enter the due date of the first payment in the current year</string>
                  </property>
                </widget>
              </item>
              <item row="8" column="0">
                <widget class="QLabel" name="textLabel4">
                  <property name="text">
                    <string>Type of interest rate</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="8" column="1">
                <widget class="KComboBox" name="m_interestType">
                  <item>
                    <property name="text">
                      <string comment="@item type of interest">Fixed</string>
                    </property>
                  </item>
                  <item>
                    <property name="text">
                      <string comment="@item type of interest">Variable</string>
                    </property>
                  </item>
                </widget>
              </item>
              <item row="9" column="0">
                <widget class="QLabel" name="textLabel3_2">
                  <property name="text">
                    <string>Time between interest changes</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="9" column="1">
                <layout class="QHBoxLayout">
                  <item>
                    <widget class="QSpinBox" name="m_interestFrequencyAmountEdit">
                      <property name="minimum">
                        <number>1</number>
                      </property>
                      <property name="maximum">
                        <number>999</number>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="KComboBox" name="m_interestFrequencyUnitEdit">
                      <property name="currentIndex">
                        <number>3</number>
                      </property>
                      <item>
                        <property name="text">
                          <string comment="@item frequency of interest calculation">Days</string>
                        </property>
                      </item>
                      <item>
                        <property name="text">
                          <string comment="@item frequency of interest calculation">Weeks</string>
                        </property>
                      </item>
                      <item>
                        <property name="text">
                          <string comment="@item frequency of interest calculation">Months</string>
                        </property>
                      </item>
                      <item>
                        <property name="text">
                          <string comment="@item frequency of interest calculation">Years</string>
                        </property>
                      </item>
                    </widget>
                  </item>
                </layout>
              </item>
              <item row="10" column="0">
                <widget class="QLabel" name="textLabel4_2">
                  <property name="text">
                    <string>Next interest change is due</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="10" column="1">
                <widget class="KMyMoneyDateInput" name="m_interestChangeDateEdit">
                  <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="spacer1">
              <property name="sizeHint">
                <size>
                  <width>29</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="spacer2">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">kcombobox.h</header>
    </customwidget>
  <customwidget>
   <class>QSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>                                                                                 
   <class>KMyMoneyPayeeCombo</class>                                                             
   <extends>QWidget</extends>                                                                    
   <header>kmymoneypayeecombo.h</header>
   <container>1</container>                                                                      
  </customwidget>        
  <customwidget>
   <class>KMyMoneyFrequencyCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneyfrequencycombo.h</header>
   <container>1</container>
  </customwidget>

    <customwidget>
      <class>AmountEdit</class>
      <extends>QLineEdit</extends>
      <header location="global">amountedit.h</header>
    </customwidget>
  </customwidgets>
</ui>
